### PR TITLE
Accessibility list check and improvements

### PR DESCRIPTION
**Pull request by LuisDavidRodriguez**
# Peer to peer Erik and David Accessibility changes 
## List all points that you have checked:
- **Page titles.** Enhance add icon in tab title.
- **Image text alternatives** A more descriptive Alt attribute added in the stories image 
- **Text headings** Not issues found 
- **Color contrast** Not issues found
- **Resize**  Not issues found, the text grow in the break point of 768px
- **Interaction** Keyboard iteration add tab to the pro, Labels added to the form and set as hidden to match the Figma template. 
> (labels are optional but recommended as voiceover tools can read them, and should be set as invisible to match the Figma design)
- **Moving content** Not issues found
- **Multimedia** Not issues found
- **The basic structure of the page** Not issues found

> If everything was ok, make a statement about that (e.g. "Page titles - no issues found").
 If you needed to change something, link to the corresponding line of code in your pull request changes.

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -246,8 +243,11 @@ <h2>Contact me</h2>
...
         <p>I'm always interested in hearing about new projects, so if you'd like touch at please get in touch.</p>
         <form action="https://formspree.io/f/mjvlqgdz" method="post">
+          <label for="name">Full Name:</label>
           <input id="name" name="Name" type="text" maxlength="30" placeholder="Full name" required/>
+          <label for="email">Email address:</label>
           <input id="email" name="email" type="email" pattern=".+@(hotmail.com|gmail.com|yahoo.com)" placeholder="Email address" required/>
+          <label for="comments">Comments:</label>
           <textarea id="comments" name="comments" cols="10" rows="5" maxlength="500" placeholder="Enter text here" required></textarea>
           <button id="btn-sub" class="button" type="submit">Get in touch</button>
         </form>
```
Comment (HAMMAS-SALEEM) [inline index.html:253]: - [ ] Your form input fields should have labels for the accessibility purpose :+1:

File: index.html [hunk 2]
```diff
@@ -168,7 +165,7 @@ <h2 class="project-subtitle">RPM Governor</h2>
           <a href="#" class="button">See Project</a>
         </article>
```
Comment (HAMMAS-SALEEM) [inline index.html:166]: ### Your buttons
buttons

- [ ] Your buttons are lacking the 4.5:1 contrast. Please fix it :+1:

File: index.html [hunk 3]
```diff
@@ -4,7 +4,8 @@
     <meta charset="UTF-8" />
     <meta name="viewport" content="width=device-width, initial-scale=1" />
     <link rel="stylesheet" href="styles.css" />
-    <title>Portfolio Luis David</title>
+    <title>Portfolio-Luis David</title>
+    <link rel="icon" href="images/favicon.ico" type="image/x-icon">
   </head>
 
   <body>
```
Comment (HAMMAS-SALEEM) [inline index.html:11]: ### Your paragraph
paragraph

- [ ] Please make your paragraph bold for the accessibility purpose :+1:

**Review by HAMMAS-SALEEM — CHANGES_REQUESTED** [review]
# Required Changes :recycle:
almost-there

### Hey Team (@LuisDavidRodriguez,  @ErikStoupignan) :wave:

### Highlights :1st_place_medal:

:heavy_check_mark: Following Linters 
:heavy_check_mark: Following GitHub Flow
:heavy_check_mark: Overall structure is good


Good job on implementing some of the requirements but there are some issues that you still need to work on to go to the next project but you are almost there! :blush:

# Optional suggestions


_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding :clap: :clap: :clap:

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me @HAMMAS-SALEEM in your question so I can receive the notification.

_**Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise.**_

------------

_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: index.html (hunk 3 above)
Comment (LuisDavidRodriguez) [reply index.html:11]: solved

File: index.html [hunk 4]
```diff
@@ -50,11 +51,7 @@ <h2>My Recent Works</h2>
       </div>
```
[the remark this answers is not in the document]
Comment (LuisDavidRodriguez) [reply index.html:52]: Solved

File: index.html (hunk 2 above)
Comment (LuisDavidRodriguez) [reply index.html:166]: Solved, changing the color of the buttons to a darker one. The test now show 4.86

File: index.html (hunk 1 above)
Comment (LuisDavidRodriguez) [reply index.html:253]: Solved, change the visibility to show

**Review by vickymarz — APPROVED** [review]
Hi @LuisDavidRodriguez , 

# 🟢  Status: APPROVED 🏆 🎉 🎊 

You have done a great job on this project, keep it up 👍 🥇. Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
congratulations!  🎉💯🌟 

## Top highlight 🎯

- The Pull Request is following the GitHub Flow ✅ 
- The PR title and description are professional and descriptive ✅
- Linters are working properly ✅ 


## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**